cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinMapper.java

```diff
@@ -157,6 +159,39 @@ private EntityDescription generateEntityDescription() {
             .collect(Collectors.toList());
     }
 
+    private List<URI> generateNvaHrcsCategoriesAndActivities() {
+        if (extractCristinHrcsCategoriesAndActivities() == null) {
```

Comment:
isNull(extractCristinHrcsCategoriesAndActivities()), or return an optional from the method called